markdown/Post-ProcessingGuides/SimpleTechniquesToFixColorBanding.md

```diff
@@ -0,0 +1,224 @@
+
+To break this down for Lightroom. 
+
+1. [example](https://i.imgur.com/YzWh5Cv.png) when Lightroom selected entire background. 
+2. [example](https://i.imgur.com/b9gOcZ0.png) when I used Subtract to Mask with a Brush to clean the areas so I am left with clean background selected only.
+3. [example](https://i.imgur.com/vVFDAxu.png) when I used a brush again to remove the background selection to focus only on color banding area. 
+4. Now you can apply the same edits as mentioned above to remove the banding.
+
+<br>  
+
+
+
+If the banding occurs duo to some lightning that bleeds out behind the subject, you can use Quick Selection Tool from Photoshop to select the banding area. Selection example in Photoshop should look like this [this](https://i.imgur.com/LI6uogy.png). Applied edits to remove the banding will only cover that area.
+
+
+In Lightroom, go to Masking (Shift+W) and select a Subject so the AI makes the selection for you. If there are two subjects and banding covers only one, then what you do is Subtract to Mask with a Brush to remove the selection from the other subject.
+Final selection example after doing the Subtract to Mask with a Brush in Lightroom looks like [this](https://i.imgur.com/Cu2SQYJ.png). Applied edits to remove the banding will only cover that area.
+
+
+<img src="../Images/PostProcessingGuides/qbbeforeandafterexample.png" /><div class="figure"><p>Smoothing out the banding that comes from the lighting which bleeds out behind.</p></div><br> 
+
+
+##Grain usage on shots in general, and why grain, noise and blur
+<p>&nbsp;</p>
+
+<left> <b><font size=+2 face="Galano Grotesque Alt">THEY CAN SOLVE COLOR BANDING</font></b> <br>
+
+
+
+@alert neutral
+Applying grain to a shot comes down to a personal choice. Its entirely your own individual preference, though I highly recommend it. Adding noise or grain can significantly improve your images.
+@end
+
+
+One of the main fighters against color banding is the grain. As seen above, a bit of grain can help to blend those harsh transitions between the tones. Grain not only that can help to eliminate the color banding it can also add a unique special feel and mood to an entire shot, or it can even determine your style. A touch of grain on your shot can add that old times feel by adding the vintage look to a shot, it can add more details, and it can make a photo look much deeper.
+
+I, personally, dont apply grain when I do processing with ReShade. I like to do it after. 
+Simply put, I want to have the cleanest image possible for various tweaks such as color adjustments etc. that I do in Lightroom. I don't want to deal with a noisy image in post, unless you are already satisfied with the shot result which has a grain on it which was applied from ReShade.<br> 
```

Comment:
We could maybe mention how, if you are planning on editing outside of Reshade (and maybe even inside the game), to get rid of the game's vanilla grain. Tools often come with that option, but most of the time it can also be done with shader toggler.

Reply:
Riiight, but after the first section is cleaned. I'm also not familiar with shader toggler. I mean I know what that is, but honestly I never used it. I know its very useful to get rid of some smokes etc. or HUD even iirc

Reply:
As with the point 4 I answered above, just mentioning it in a sentence would be enough. I would totally recommend it to check it out tho, lets you toggle off a lot of unwanted stuff in games, that being objects or effects, such as grain.

Reply:
But good mention about getting rid of vanilla grain. Its what bothers me as well tbh. in some games, ahhh, and here I thought I was done with the guide! This is kinda important to mention. I mean my guide focuses on how to fix color banding, but this can be mentioned, like sneaky mention. The only thing is that I never did this with shader toggler, but I heared before about it. Maybe it was from some discussions from the Flickr, I can't remember really

Reply:
We probably sent the message at the same time lol. Yep 4:45